testing/collect_tests.py

```diff
@@ -206,6 +204,17 @@ def run_example(topdir, generator, ci_environment, buildflags, recipe, example):
             return_code += run_command(
                 step=step, command=command, expect_failure=expect_failure)
 
+        # execute dashboard script, if it exists
+        dashboard_script = 'dashboard.cmake'
+        dashboard_script_path = cmakelists_path / dashboard_script
+        if dashboard_script_path.exists():
+            # if this directory contains a dashboard.cmake script, we launch it
+            step = dashboard_script
+            command = r'ctest -C {0} -S {1} -DCTEST_CMAKE_GENERATOR={2}'.format(
+                configuration, dashboard_script_path, generator)
```

Comment:
Based on Appveyor output it seems that `dashboard_script_path` is not properly understood and I think it may need to be converted to `str`.

Reply:
It finds it allright, it's passing the generator that makes trouble: 
```
'ctest', '-C', 'Debug', '-S', 'C:projectscmake-cookbook-no-symlinkschapter-14recipe-03cxx-exampledashboard.cmake', '-DCTEST_CMAKE_GENERATOR=MSYS', 'Makefiles'
```

Reply:
Sorry, both our analyses were correct. Let's see what my last commit does.